helper/src/components/deployTab.js

```diff
@@ -75,6 +75,7 @@ export default function DeployTab({ defaults, updateFn, tabValues, invalidArray,
       })
     }),
     ...(addons.csisecret !== "none" && { azureKeyvaultSecretsProvider: true }),
+    ...(addons.kvPollInterval !== "2m" && { kvPollInterval: addons.kvPollInterval }),
```

Comment:
You don't need to hardcode 2m here. You can just refer to the defaults variable.

```javascript
  ...(addons.kvPollInterval !== defaults.addons.kvPollInterval  && { kvPollInterval: addons.kvPollInterval }),
```

Reply:
Yea, we only want to send the "kvPollInterval" if csisecret !== 'none'.   This is if the user selects csisecret, sets an interval, then changes their mind & says none again.  So:

```...(addons.csisecret !== "none" && addons.kvPollInterval !== defaults.addons.kvPollInterval  && { kvPollInterval: addons.kvPollInterval }),```

Reply:
@liammoat - Are you ok to make this update too...?  Then i think we're good to merge. 🚀

Reply:
@Gordonby - Yep, I'll update today.